android/sdk/build.gradle

```diff
@@ -52,24 +52,20 @@ gradle.projectsEvaluated {
         def buildNameCapitalized = "${buildType.name.capitalize()}"
         def bundlePath = "${buildDir}/intermediates/bundles/${buildType.name}"
 
-        // Bundle fonts in react-native-vector-icons.
-        //
+        // Copy fonts from react-native-vector-icons and /fonts to assets folder.
 
         def currentFontTask = tasks.create(
                 name: "copy${buildNameCapitalized}Fonts",
                 type: Copy) {
             from("${projectDir}/../../fonts/jitsi.ttf")
             from("${projectDir}/../../node_modules/react-native-vector-icons/Fonts/")
-            into("${bundlePath}/assets/fonts")
+            into("${projectDir}/src/main/assets/fonts")
         }
 
-        currentFontTask.dependsOn("merge${buildNameCapitalized}Resources")
-        currentFontTask.dependsOn("merge${buildNameCapitalized}Assets")
+        runBefore("generate${buildNameCapitalized}Resources", currentFontTask)
+        runBefore("generate${buildNameCapitalized}Assets", currentFontTask)
```

Comment:
Assets are easy without copying to android/sdk/src/main/assets and .gitignore-ing then:
1) Change `android.buildTypes.all { buildType ->` above to `android.libraryVariants.all { variant ->`
2) Add `def buildType = variant.buildType`
3) Change `into("${projectDir}/src/main/assets/fonts")` to `into("${variant.mergeAssets.outputDir}/fonts")`
4) Change `currentFontTask.dependsOn("merge${buildNameCapitalized}Assets")` to `currentFontTask.dependsOn(variant.mergeAssets)`. I'm not exactly sure why you deleted the mergeAssets dependency. The mergeResources dependencies doesn't make sense to me because the currentFontTask produces assets only, not resources.
5) Change `runBefore("generate${buildNameCapitalized}Assets", currentFontTask)` to `variant.outputs.each { output -> output.processResources.dependsOn(currentFontTask) }`

Which eventually results in pretty much the old source code without figuring out what `process${buildNameCapitalized}Resources` is called now.

Reply:
Wow, didn't know you know Gradle this much... should have just handed over the task to you ;) Thanks, this works perfectly!

Reply:
Wow, didn't know you know Gradle this much... should have just handed over the task to you ;) Thanks, this works perfectly!